packages/tasks/src/tasks/sentence-similarity/data.ts

```diff
@@ -94,7 +94,7 @@ const taskData: TaskDataCustom = {
 	],
 	summary:
 		"Sentence Similarity is the task of determining how similar two texts are. Sentence similarity models convert input texts into vectors (embeddings) that capture semantic information and calculate how close (similar) they are between them. This task is particularly useful for information retrieval and clustering/grouping.",
-	widgetModels: ["sentence-transformers/all-MiniLM-L6-v2"],
+	widgetModels: ["thenlper/gte-large"],
 	youtubeId: "VCZq5AkbNEU",
```

Comment:
cc @tomaarsen in case you think we should highlight others

Reply:
If we want to keep using a small model, then I think we should use https://huggingface.co/BAAI/bge-small-en-v1.5

Otherwise likely https://huggingface.co/BAAI/bge-large-en-v1.5 or https://huggingface.co/mixedbread-ai/mxbai-embed-large-v1
gte-large is a bit dated, and some other options (gte-large-v1.5, nomic-v1.5) require `trust_remote_code` which we should probably avoid here.